types/dialogflow/index.d.ts

```diff
@@ -892,6 +958,20 @@ export type MatchMode =
   | "MATCH_MODE_HYBRID"
   | "MATCH_MODE_ML_ONLY";
 
+// API version for the agent.
+export type ApiVersion =
+  | 'API_VERSION_UNSPECIFIED'
```

Comment:
Please keep quotation marks consistent with the code - this project follows the TypeScript coding guidelines, using double quotes for strings.

https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#strings